Transaction.cs

```diff
@@ -0,0 +1,38 @@
+﻿public class Transaction
+{
+    private string date;
+    public string from;
+    public string to;
+    private string narrative;
+    private decimal amount;
+
+    public Transaction(string[] fields)
```

Comment:
This makes it hard to identify which fields this constructor actually requires. You should need to read the code to find out what you need to pass to a method, the method signature should tell you that 🙂